khelo/lib/ui/flow/settings/edit_profile/edit_profile_screen.dart

```diff
@@ -54,7 +54,7 @@ class EditProfileScreen extends ConsumerWidget {
               fit: BoxFit.contain,
               colorFilter: ColorFilter.mode(
                   state.isButtonEnable && !state.isImageUploading
-                      ? context.colorScheme.textPrimary
+                      ? context.colorScheme.primary
```

Comment:
@cp-mayank we have changed actionButton to textPrimary color after discussing with digita, revert this change.